interface/src/app/mlmodel/validator.tpl.html

```diff
@@ -0,0 +1,5 @@
+<div class="alert alert-danger" role="alert" ng-if="$ctrl.errMessage">
+  {{$ctrl.errMessage}}
+  <br>
+  Please click <a href="#/home">Home</a> to reset it.
```

Comment:
@mhuyck: I discussed this with @cgreene and we actually did this on purpose. Please feel free to file an issue. We can discuss further when I am back from vacation. Thanks.